### DISCUSSION TITLE
Use subprocess module to execute python orders from inside PyInstaller

### DISCUSSION THREAD

**Discussion by esantamariavazquez** [pyinstaller/pyinstaller#6636] [Help] [answered]
Hello,

I would like to know if it's possible to use the python bundled within a PyInstaller executable to execute commands from the command line using subprocess module. For example, I need to create a virtual environment to install additional dependencies depending on several criteria by executing the following line:

```
subprocess.Popen('python -m venv venv', shell=True,
                              stdout=subprocess.PIPE,
                              stderr=subprocess.PIPE,
                              universal_newlines=True)
```

I know that it seems a weird thing to do, but it's necessary for my use case.

Thank you in advance!! Keep going with this project, it's awsome!

**Comment by rokm** [comment, marked as answer]
Nope. Python interpreter executable is not bundled with PyInstaller, only the shared library is (which is used to launch embedded interpreter that is used to run your frozen script).
